Components/Avatar/avatar.css

```diff
@@ -0,0 +1,40 @@
+.avatar {
+  border-radius: 50%;
+  padding: 2px;
+  border: 1px solid var(--violet-color);
```

Comment:
One suggestion instead of using fixed color use primary color or any common color. It will make your design more consistent.

Reply:
what do you mean by 'common color' and 'fixed color'. could you please elaborate further?

Reply:
Like use primary color for the border. So it will look consistent. If we use different colors at different places then our design will not look that much consistent.

Reply:
my primary color is violet even though i didnt name it properly. will fix the naming soon.